evap/staff/forms.py

```diff
@@ -102,6 +102,11 @@ def clean(self):
 
 
 class CourseTypeForm(forms.ModelForm):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        self.fields["order"].widget = forms.HiddenInput()
```

Comment:
i see we use these `HiddenInput`s a lot, but is there a reason why we can't just exclude the field from `fields`?

Reply:
the field must be included in the form to post the new value after reordering and saving the form, doesn't it?

Reply:
oh, i didn't realize that this is the form used on the course type index page.